operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md

```diff
@@ -0,0 +1,198 @@
+    ```bash
+    ncn-m001# ./utils/secrets-decrypt.sh cray_hms_rts_credentials ./certs/sealed_secrets.key ./customizations.yaml | jq .data.vault_rts_defaults -r | base64 -d | jq
+    {
+      "Username": "root",
+      "Password": "secret"
+    }
+    ```
+
+2.  Update the default credentials in `customizations.yaml` for RTS:
+    
+    Specify the desired default ServerTech PDU credentials:
+    ```bash
+    ncn-m001# echo '{"Username":"admn", "Password":"foobar"}' | base64 > rts.pdu.creds.json.b64
+    ```
+
+    Specify the desired default RTS Redfish interface credentials:
+    ```bash
+    ncn-m001# echo '{"Username":"root", "Password":"supersecert"}' | base64 > rts.redfish.creds.json.b64
+    ```
+
+
+    Update and regenerate `cray_hms_rts_credentials` sealed secret:
+    ```bash
+    ncn-m001# cat << EOF | yq w - 'data.vault_pdu_defaults' "$(<rts.pdu.creds.json.b64)" | yq w - 'data.vault_rts_defaults' "$(<rts.redfish.creds.json.b64)" | yq r -j - | ./utils/secrets-encrypt.sh | yq w -f - -i ./customizations.yaml 'spec.kubernetes.sealed_secrets.cray_hms_rts_credentials'
+    {
+        "kind": "Secret",
+        "apiVersion": "v1",
+        "metadata": {
+            "name": "cray-hms-rts-credentials",
+            "namespace": "services",
+            "creationTimestamp": null
+        },
+        "data": {}
+    }
+    EOF
+    ```
+
+4.  Decrypt generated secret for review.
+    
+    Default ServerTech PDU credentials:
+    ```bash
+    ncn-m001# ./utils/secrets-decrypt.sh cray_hms_rts_credentials ./certs/sealed_secrets.key ./customizations.yaml | jq .data.vault_pdu_defaults -r | base64 -d | jq
+    {
+      "Username": "admn",
+      "Password": "foobar"
+    }
+    ```
+
+    Default RTS Redfish interface credentials:
+    ```bash
+    ncn-m001# ./utils/secrets-decrypt.sh cray_hms_rts_credentials ./certs/sealed_secrets.key ./customizations.yaml | jq .data.vault_rts_defaults -r | base64 -d | jq
+    {
+      "Username": "root",
+      "Password": "supersecert"
+    }
+    ```
+
+
+5.  Update the site-init secret for the system:
+    ```bash
+    ncn-m001# kubectl delete secret -n loftsman site-init
+    ncn-m001# kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml
+    ```
+
+#### 1.3 Redeploy RTS to pick up the new sealed secret and push credentials into vault.
+1.  Determine the version of RTS:
+    ```bash
+    ncn-m001# RTS_VERSION=$(kubectl -n loftsman get cm loftsman-core-services -o jsonpath='{.data.manifest\.yaml}' | yq r - 'spec.charts.(name==cray-hms-rts).version')
+    ncn-m001# echo $RTS_VERSION
+    ```
+
+2.  Create `rts-manifest.yaml`:
+    ```bash
+    ncn-m001# cat > rts-manifest.yaml << EOF 
+    apiVersion: manifests/v1beta1
+    metadata:
+        name: rts
+    spec:
+        charts:
+        - name: cray-hms-rts
+          version: $RTS_VERSION
+          namespace: services
+    EOF
+    ```
+
+2.  Merge customizations.yaml with rts-manifest.yaml
```

Comment:
Missing period

Reply:
Same comment as above about enclosing file names with backticks

Reply:
Should be resolved fixed now.